packages/notification-center/rollup.config.js

```diff
@@ -32,6 +34,29 @@ export default [
       image(),
     ],
   },
+  {
+    input: 'src/web-component.ts',
+    external: ['react@17.0.2', 'react-dom@17.0.2'],
+    output: [
+      {
+        file: packageJson.browser,
+        format: 'umd',
+        name: 'NotificationCenterWebComponent',
+        sourcemap: true,
+      },
+    ],
+    plugins: [
+      replace({
+        'process.env.NODE_ENV': JSON.stringify('production'),
+      }),
```

Comment:
the react and other libs are using `process.env.NODE_ENV` variable, by default Rollup doesn't include it